### ISSUE TITLE
Default widgets should be providing good accessibility defaults

### ISSUE THREAD

**Issue by vvo** [algolia/instantsearch.js#1301]
We need to ensure before releasing that we are good in term of accessibility (A11Y).

Funny eslint plugin: https://github.com/evcohen/eslint-plugin-jsx-a11y

We will need help from anyone skilled in a11y or just learn it.

**Comment by mthuret** [comment]
React InstantSearch has been moved here: https://github.com/algolia/react-instantsearch
This issue has either been rewritten or placed inside our backlog.
